scipp-developer.yml

```diff
@@ -43,6 +43,8 @@ dependencies:
   - pre-commit
   - yapf
   - flake8
+  - pip:
+    - nb-clean
```

Comment:
Now CI will install this twice?

Reply:
Yes. Just like flake8 and yapf. Is there a way around it?

Reply:
I guess the issue is that we use these `yml` files for CI *and* developers? Maybe with @nvaytet's script we can include the non-CI requirements as extras when making envs for devs?